db/data_migration/20200311133826_make_command_paper_numbers_valid.rb

```diff
@@ -0,0 +1,12 @@
+Attachment
+  .where.not(command_paper_number: nil)
+  .where.not(command_paper_number: "")
+  .each do |attachment|
+  valid_number = attachment.command_paper_number # for example, "CM.123-iv "
+    .tr(". ", "") # remove periods and spaces: "CM123-iv"
+    .gsub(/(\d+)/) { |number| ". #{number}" } # inject period & space before number: "CM. 123-iv"
```

Comment:
I think we may need `sub` specifically here, as this could also match subsequent numbers e.g. `CM 123-1`.

Reply:
Good point. I think there's only one case where we have a number suffix, which content designers will fix manually, but still better to be safe here.